src/comp/comp.py

```diff
@@ -24,48 +24,50 @@ def __repr__(self):
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with 'D':
 print("Starts with D:")
-a = []
-print(a)
+a = [h.name.lower().strip("Alice Bob Charlie Eve Frank Glenn Harrison Igon").title() for h in humans]
```

Comment:
here you are hard coding the names and trying to cut out the rest. Here  is are a few examples of ways to do this: 

"a = [h.name for h in humans if h.name[0] == 'D']"

"a = [h.name for h in humans if h.name.startswith("D")]

You would need to go back and fix all of the ones that have strip() so that the syntax is correct.